OverlayPlugin.Core/NetworkProcessors/LineSpawnObject.cs

```diff
@@ -0,0 +1,110 @@
+            public uint layoutId;
+            public uint contentId;
+            public uint ownerId;
+            public uint bindLayoutId; // Other sources call this `gimmickId` as well
+            public float scale;
+            public ushort sharedGroupTimelineState;
+            public ushort rotation;
+            public ushort fate;
+            public byte permissionInvisibility; // According to Sapphire. No clue what this actually means
+            public byte args1;
+            public uint args2;
+            public uint args3;
+            public float x;
+            public float y;
+            public float z;
+            public uint padding2;
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                return $"{ActorID:X8}|{index:X2}|{kind:X2}|{flags:X2}|" +
+                    $"{baseId:X8}|{entityId:X8}|{layoutId:X8}|{contentId:X8}|{ownerId:X8}|{bindLayoutId:X8}|" +
+                    $"{scale:F4}|{sharedGroupTimelineState:X4}|{FFXIVRepository.ConvertHeading(rotation):F4}|{fate:X4}|" +
+                    $"{permissionInvisibility:X2}|{args1:X2}|{args2:X8}|{args3:X8}|" +
+                    // Y and Z coordinates inverted to match other lines
+                    $"{x:F4}|{z:F4}|{y:F4}";
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        public unsafe struct SpawnObject_v700 : IPacketStruct
+        {
+            public byte index;
+            public byte kind;
+            public byte flags;
+            public byte padding1;
+            public uint baseId;
+            public uint entityId;
+            public uint layoutId;
+            public uint contentId;
+            public uint ownerId;
+            public uint bindLayoutId; // Other sources call this `gimmickId` as well
+            public float scale;
+            public ushort sharedGroupTimelineState;
+            public ushort rotation;
+            public ushort fate;
+            public byte permissionInvisibility; // According to Sapphire. No clue what this actually means
+            public byte args1;
+            public uint args2;
+            public uint args3;
+            // This padding was moved in 7.0, probably due to compiler changes?
+            public uint padding2;
+            public float x;
+            public float y;
+            public float z;
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                return $"{ActorID:X8}|{index:X2}|{kind:X2}|{flags:X2}|" +
+                    $"{baseId:X8}|{entityId:X8}|{layoutId:X8}|{contentId:X8}|{ownerId:X8}|{bindLayoutId:X8}|" +
+                    $"{scale:F4}|{sharedGroupTimelineState:X4}|{FFXIVRepository.ConvertHeading(rotation):F4}|{fate:X4}|" +
+                    $"{permissionInvisibility:X2}|{args1:X2}|{args2:X8}|{args3:X8}|" +
+                    // Y and Z coordinates inverted to match other lines
+                    $"{x:F4}|{z:F4}|{y:F4}";
+            }
```

Comment:
Misc. thoughts, feel free to take or leave:
* At least in the examples given, it looks like `entityId` is the same as the `ActorID` from the packet header.  Maybe drop this as duplicative? Although if there are (perhaps-as-yet-unknown) scenarios where they could be different, I'm fine leaving it in.
* I think we can probably drop `contentId`?
* Any reason `args2` and `args3` are `:X8` when args1 is `:X2`?

FWIW, with some exceptions, my personal preference is toward using `:X` rather than fixed-length on most fields.  It helps avoid really lengthy loglines with lots of zero-padded values that look like raw packet data instead of extracted values.  It's also easier for downstream consumers to extract the values directly rather than always having to strip leading zeros.  But just a personal preference; ymmv.

(Same comments for the `_v655` struct.)